### DISCUSSION TITLE
Scroll bar missing when option "Show Pages Continuosly" is turned off

### DISCUSSION THREAD

**Discussion by user548112** [sumatrapdfreader/sumatrapdf#2713] [Q&A]
Hi,

SumatraPDF is a great reader but I'm missing one thing... I have "Show Pages Continuosly" turned off, but when this is inactive the scroll bar is missing.

I have read some other topics on this but I have PDF files which have 1000+ pages and a scrollbar would be the last thing I needed.

I know that there are keyboard shortcuts but I don't when to use my keyboard, just only the mouse.

Am I missing something here?

**Comment by GitHubRulesOK** [comment]
scrollbar on/off is available in mouse right click 
- EXCEPT there are no scrollbars needed in a single numbered page view (no need to scroll width or height)
-  In previews or fixed width only a vertical bar is visible (no horizontal) when needed with more than one screen high
- The number indicator will show page progress 
- in page by page presentation mode like fullscreen you can use info to view the current page progress number info
